Client.xcodeproj/xcshareddata/xcschemes/Fennec_Enterprise_XCUITests.xcscheme

```diff
@@ -20,175 +20,164 @@
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "E4D567211ADECE2700F1EFE7"
                BuildableName = "ReadingListTests.xctest"
                BlueprintName = "ReadingListTests"
                ReferencedContainer = "container:Client.xcodeproj">
             </BuildableReference>
-         </TestableReference>
-         <TestableReference
-            skipped = "NO">
-            <BuildableReference
-               BuildableIdentifier = "primary"
-               BlueprintIdentifier = "3BFE4B061D342FB800DDF53F"
-               BuildableName = "XCUITests.xctest"
-               BlueprintName = "XCUITests"
-               ReferencedContainer = "container:Client.xcodeproj">
-            </BuildableReference>
-            <SkippedTests>
-               <Test
-                  Identifier = "SiteLoadTest">
-               </Test>
-            </SkippedTests>
-         </TestableReference>
-         <TestableReference
-            skipped = "YES">
+         </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "NO"
+            buildForRunning = "NO"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "D39FA15E1A83E0EC00EE869C"
                BuildableName = "UITests.xctest"
                BlueprintName = "UITests"
                ReferencedContainer = "container:Client.xcodeproj">
             </BuildableReference>
-            <SkippedTests>
-               <Test
-                  Identifier = "BookmarkingTests">
-               </Test>
-               <Test
-                  Identifier = "FindInPageTests">
-               </Test>
-               <Test
-                  Identifier = "HomePageSettingsUITests">
-               </Test>
-               <Test
-                  Identifier = "NavigationTests">
-               </Test>
-               <Test
-                  Identifier = "PrivateBrowsingTests">
-               </Test>
-               <Test
-                  Identifier = "ReaderViewUITests">
-               </Test>
-               <Test
-                  Identifier = "SearchSettingsUITests">
-               </Test>
-               <Test
-                  Identifier = "SearchTests">
-               </Test>
-               <Test
-                  Identifier = "SettingsTests">
-               </Test>
-               <Test
-                  Identifier = "TopTabsTests">
-               </Test>
-               <Test
-                  Identifier = "ViewMemoryLeakTests">
-               </Test>
-            </SkippedTests>
-         </TestableReference>
-         <TestableReference
-            skipped = "YES">
```

Comment:
Did you mean to stop skipping these tests?

Reply:
yeah no tests are skipped (above shows that I removed the skipped designation for some reason) - not sure why it set it as a diff though.